kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/index.vue

```diff
@@ -290,6 +333,11 @@
           });
       }
     },
+    created() {
+      this.getBookmarks().then(() => {
+        this.getBookmarksData();
+      });
+    },
```

Comment:
There is room for optimization here as we're hitting the DB two separate times when we could get the data in a single query. If this isn't in scope for this PR then we can tackle it in a follow-up issue. I wish I'd thought of it when working on #8095